option optimize_for = LITE_RUNTIME;

message CommonAnswer
{
 required uint32 result = 1;
}
